src/proxy_lib/proxy_lib.h

```diff
@@ -14,6 +14,7 @@ extern "C" {
 
 void proxy_lib_create_common(void);
 void proxy_lib_destroy_common(void);
+int proxy_lib_is_from_UMF(void *ptr);
```

Comment:
How will this API be used and why can the client not use the `umfPoolByPtr()` function from UMF?

Reply:
It is not needed, see: https://github.com/oneapi-src/unified-memory-framework/pull/681#issuecomment-2324149846

Reply:
This PR had been closed.